imagetest/test_suites/metadata/metadata_utils.go

```diff
@@ -4,44 +4,11 @@ import (
 	"fmt"
 	"io"
 	"os/exec"
-	"strings"
 	"time"
 
 	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
 )
 
-// yumDnfRunPackageRepoQuery is a common utility between yum and dnf to run the query command
-// provided in the cmd argument and parses its output. The output format is the same both for
-// yum and dnf (hence the common utility code).
-func yumDnfRunPackageRepoQuery(cmd *exec.Cmd) (string, error) {
-	output, err := cmd.CombinedOutput()
-	if err != nil {
-		return "", fmt.Errorf("failed to run package manager command: %v, output: %s", err, string(output))
-	}
-
-	for _, line := range strings.Split(string(output), "\n") {
-		if strings.HasPrefix(line, "From repo") {
-			fields := strings.Fields(line)
-			if len(fields) != 4 {
-				return "", fmt.Errorf("invalid \"From repo\" line, got %d fields, expected 4", len(fields))
-			}
-			return fields[3], nil
-		}
-	}
-
-	return "", nil
-}
-
-// yumGetPackageRepo queries the yum package database for the repository pkg was installed from.
-func yumGetPackageRepo(pkg string) (string, error) {
-	return yumDnfRunPackageRepoQuery(exec.Command("yum", "info", pkg))
-}
-
-// dnfGetPackageRepo queries the dnf package database for the repository pkg was installed from.
-func dnfGetPackageRepo(pkg string) (string, error) {
-	return yumDnfRunPackageRepoQuery(exec.Command("dnf", "info", "-C", "--installed", pkg))
-}
-
 func reinstallPackage(pkg string) error {
```

Comment:
This isn't really a generic function anymore (which is fine) but it doesn't need the argument since it's only really capable of reinstall a few packges and we only use it once. Can we drop `pkg string`, rename it `reinstallAgent` and hardcode `google-compute-engine-windows`/`google-guest-agent` package names?

Reply:
done. thank you.